code/Attack.ipynb

```diff
@@ -0,0 +1,1471 @@
+{
```

Comment:
<p>Which method was used for generating the adversarial examples? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sayakpaul/robustness-foundation-models/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>